js/glowingbear.js

```diff
@@ -66,6 +66,12 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
     });
     $scope.settings = settings;
 
+    //For upgrade reasons because we changed the name of host to hostField
+    //check if the value might still be in the host key instead of the hostField key
+    if (!settings.hostFieldv && settings.host) {
```

Comment:
there's a stray `v` here in `hostFieldv`

Reply:
does this need to trigger `parseHost`?

Reply:
I don't think it does because it's already triggered at init and this is in the constructor before init. Thanks for fixing the typo and the merge.

Reply:
Yeah it worked fine in testing